scripts/install/jellyfin.sh

```diff
@@ -116,234 +51,51 @@ cat > "/home/${username}/.config/Jellyfin/config/dlna.xml" <<-CONFIG
-  <ImageExtractionTimeoutMs>0</ImageExtractionTimeoutMs>
-  <PathSubstitutions />
-  <EnableSimpleArtistDetection>true</EnableSimpleArtistDetection>
-  <UninstalledPlugins />
+  <RequireHttps>true</RequireHttps>
 </ServerConfiguration>
 CONFIG
 #
-# Create the service file that will start and stop jellyfin.
-cat > "/etc/systemd/system/jellyfin.service" <<-SERVICE
-[Unit]
-Description=Jellyfin
-After=network.target
-
-[Service]
-User=${username}
-Group=${username}
-UMask=002
-
-Type=simple
-WorkingDirectory=$install_dir
-ExecStart=$install_dir/jellyfin -d /home/${username}/.config/Jellyfin
-TimeoutStopSec=20
-KillMode=process
-Restart=always
-RestartSec=2
-
-[Install]
-WantedBy=multi-user.target
-SERVICE
+# Add the jellyfin official repository and key to our installation so we can use apt-get to install it jellyfin and jellyfin-ffmepg.
+wget -q -O - "https://repo.jellyfin.org/$DIST_ID/jellyfin_team.gpg.key" | apt-key add - >> "${log}" 2>&1
+echo "deb [arch=$( dpkg --print-architecture )] https://repo.jellyfin.org/$DIST_ID $DIST_CODENAME main" > /etc/apt/sources.list.d/jellyfin.list
+#
+# install jellyfin and jellyfin-ffmepg using apt functions.
+apt_update #forces apt refresh
+apt_install jellyfin jellyfin-ffmpeg
+#
+# Add the jellyfin user to the master user's group.
+usermod -a -G "${username}" jellyfin
+#
+chown jellyfin:jellyfin /etc/jellyfin/dlna.xml
+chown jellyfin:jellyfin /etc/jellyfin/system.xml
+chown jellyfin:root /etc/jellyfin/logging.json
+chown jellyfin:adm /etc/jellyfin
 #
 # Configure the nginx proxypass using positional parameters.
 if [[ -f /install/.nginx.lock ]]; then
-    bash "/usr/local/bin/swizzin/nginx/jellyfin.sh" "${app_port_http}" "${app_port_https}"
-    systemctl reload nginx
+	bash /usr/local/bin/swizzin/nginx/jellyfin.sh
+	systemctl -q restart nginx.service
 fi
-echo_progress_done "Jellyfin configured"
 #
-# Set the correct and required permissions of any directories we created or modified.
-chown "${username}.${username}" -R "$install_dir"
-chown "${username}.${username}" -R "$install_ffmpeg"
-chown "${username}.${username}" -R "/home/${username}/.config"
-chown "${username}.${username}" -R "/home/${username}/.ssl"
-#
-# Enable and start the jellyfin service.
-systemctl daemon-reload -q
-systemctl enable -q --now "jellyfin.service" 2>&1  | tee -a $log
+# Restart the jellyfin service to make sure our changes take effect
+systemctl -q restart "jellyfin.service"
 #
 # This file is created after installation to prevent reinstalling. You will need to remove the app first which deletes this file.
-touch "/install/.jellyfin.lock"
-#
-echo_success "Jellyfin installed"
+touch /install/.jellyfin.lock
 #
-if [[ ! -f /install/.nginx.lock ]]; then
-    echo_info "Jellyfin is available at: https://$(curl -s4 icanhazip.com):${app_port_https}"
-else
-    echo_info "Jellyfin is now available in the panel"
-    echo_info "Please visit https://$ip_address/jellyfin"
-fi
+echo_success "The Jellyfin installation has completed"
```

Comment:
Should be "Jellyfin installed" to be in line with the rest

Reply:
changed to "Jellyfin installed"